src/__tests__/__snapshots__/downshift.aria.js.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic snapshot 1`] = `
+<div
+  aria-expanded="false"
+  aria-haspopup="listbox"
+  aria-labelledby="downshift-0-label"
+  aria-owns="downshift-0-menu"
+  role="combobox"
+>
+  <label
+    data-testid="label"
+    for="downshift-0-input"
+    id="downshift-0-label"
+  >
+    label
+  </label>
+  <input
+    aria-autocomplete="list"
+    aria-controls="downshift-0-menu"
+    aria-expanded="false"
+    aria-labelledby="downshift-0-label"
+    autocomplete="off"
+    data-testid="input"
+    id="downshift-0-input"
+    role="listbox"
+    value="item"
+  />
+  <button
+    aria-expanded="false"
+    aria-haspopup="true"
+    aria-label="open menu"
+    data-testid="button"
+    data-toggle="true"
+    role="button"
+    type="button"
+  />
+  <ul
+    aria-labelledby="downshift-0-label"
```

Comment:
If you wanted this label to be more explicit about what the results list included, it could be changed to an `aria-label` with a prop to let users enter their own string.

Reply:
This is a great suggestion. Now if you pass `aria-label` to `getMenuProps` we'll use that instead of `aria-labelledby`. It works perfectly :+1:

Reply:
If you want to be labelling explicitly the listbox options, which maybe contextually different than that of the overall label, what you are going to want to try to replicate semantically is a `select[multiple]` with a `optgroup` e.g.

```
<select multiple>
  <optgroup label="pets">
    <option>Cat</option>
    <option>Dog</option>
    <option>Fish</option>
  </optgroup>
</select>
```

The psuedo markup structure you want to recreate that is 

```
<div role="listbox">
  <div role="group" aria-label="Pets">
    <span role="none">Pets</span> <!-- optgroup shows a visible label, you could choose to exclude this but it's nice for a visual grouping -->
    <span role="option>Dog</span>
    <span role="option>Cat</span>
    <span role="option>Fish</span>  
  </div>
</div>
```

Reply:
Interesting! Thanks. I think this isn't the level of abstraction for that (lots of use cases for this component), but perhaps we should have an example for this 🤔